frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseGitMacheteRepositoryReadyAction.java

```diff
@@ -3,12 +3,20 @@
 import com.intellij.openapi.actionSystem.AnAction;
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.project.DumbAwareAction;
+import lombok.CustomLog;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
 
 import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeyGitMacheteRepository;
+import com.virtuslab.logger.IEnhancedLambdaLogger;
 
+@CustomLog
```

Comment:
I don't think this should be defined in abstract base class... this might lead to class name being logged incorrectly (base class name instead of concrete class name)

Reply:
It's basically the question of what class name we want to see in the logs coming from `IExpectsKey...`